packages/contracts/test/generator/GenArt721GeneratorV0.test.ts

```diff
@@ -290,7 +299,7 @@ describe(`GenArt721GeneratorV0`, async function () {
       const hash = hashes[0];
       expect(tokenHtml).to.include(
         getScriptTag(
-          `let tokenData = JSON.parse(\`{"tokenId":"0","hashes":["${hash}"]}\`, (key, value) => key === "data" && value !== null ? atob(value) : value);`
+          `let tokenData = JSON.parse(\`{"tokenId":"0","hashes":["${hash}"]}\`, (key, value) => key === "data" && value !== null ? value.startsWith("#web3call#") ? Object.entries(JSON.parse(atob(value.slice(10)))).reduce((acc, [k, v]) => ((acc[atob(k)] = atob(v)), acc), {}) : atob(value) : value);`
```

Comment:
Nit: might make sense to break out a function to generate this string so we can just update in one place if this changes again

Reply:
good call - I broke out the reviver function as a constant since it is used across several of the tests. The other variation is mostly due to certain contract-version-specific formats, so figured okay to let those stay as is. If we update reviver in future, will only be one place now 👍